app/services/withdraw_participant.rb

```diff
@@ -32,10 +32,10 @@ def call
       participant_profile.training_status_withdrawn!
     end
 
-    induction_coordinator = participant_profile.school.induction_coordinator_profiles.first
+    induction_coordinator = relevant_induction_record.school.induction_coordinator_profiles.first
     if induction_coordinator.present?
       SchoolMailer.with(
-        withdrawn_participant: participant_profile,
+        withdrawn_participant: relevant_induction_record,
```

Comment:
should we rename this param to induction record rather than withdrawn participant?